### PR TITLE
Update content-box

### PR DESCRIPTION
**Pull request by matzimowski**
closes https://github.com/brainly/style-guide/issues/628
closes https://github.com/brainly/style-guide/issues/475
closes https://github.com/brainly/style-guide/issues/452

preview:
screen shot 2016-04-14 at 15 33 28

screen shot 2016-04-15 at 10 39 54

### CODE REVIEW

**Comment by kdzwinel** [comment]
examples? :(

File: src/components/content-box/_content-box.scss [hunk 1]
```diff
@@ -38,6 +38,10 @@ $includeHtml: false !default;
     &__actions {
       @include componentsContainer;
       min-height: rhythm(2);
+
+      &--with-elements-to-right {
+        justify-content: flex-end;
```
Comment (aju) [inline src/components/content-box/_content-box.scss]: In long term should be removed - use only mint-actions-list for such cases

Comment (matzimowski) [inline src/components/content-box/_content-box.scss]: OK

**Comment by matzimowski** [comment]
@kdzwinel @aju updated preview

File: src/components/content-box/_content-box.scss [hunk 2]
```diff
@@ -52,6 +56,46 @@ $includeHtml: false !default;
     &--full {
       width: 100%;
     }
+
+    &__title,
+    &__actions {
+      &--with-centered-elements {
```
Comment (aju) [inline src/components/content-box/_content-box.scss]: preview?

File: src/components/content-box/_content-box.scss [hunk 3]
```diff
@@ -38,6 +38,10 @@ $includeHtml: false !default;
     &__actions {
       @include componentsContainer;
       min-height: rhythm(2);
+
+      &--with-elements-to-right {
```
Comment (aju) [inline src/components/content-box/_content-box.scss]: preview ?

File: src/components/content-box/_content-box.scss (hunk 2 above)
Comment (matzimowski) [inline src/components/content-box/_content-box.scss]: added

File: src/components/content-box/_content-box.scss (hunk 3 above)
Comment (matzimowski) [inline src/components/content-box/_content-box.scss]: added

**Comment by matzimowski** [comment]
@aju @kdzwinel updated previews

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "style-guide",
-  "version": "26.5.0",
+  "version": "26.6.0",
```
Comment (aju) [inline package.json]: whats that?

**Comment by aju** [comment]
OK

File: package.json (hunk 4 above)
Comment (matzimowski) [inline package.json]: what's wrong?

**Comment by kdzwinel** [comment]
ok

**Pull request merged**